debian/libjaeger.install

```diff
@@ -1,3 +1,4 @@
-#usr/lib/libopentracing.so.*
-#usr/lib/libjaegertracing.so.*
-#usr/lib/libthrift.so.*
+usr/lib/libopentracing.so.*
```

Comment:
i'd suggest fold "cmake: cleanup debian/libjaeger.install mangling" into this commit.

Reply:
https://github.com/ceph/ceph/commit/848df6d497f12d57b042b2a73d23f17130088e4e